vsintegration/packages.config

```diff
@@ -16,12 +16,14 @@
   <package id="Microsoft.VisualStudio.Threading" version="14.1.131" targetFramework="net46" />
   <package id="Microsoft.VisualStudio.CoreUtility" version="14.3.25407" targetFramework="net46" />
   <package id="Microsoft.VisualStudio.Shell.Design" version="14.3.25407" targetFramework="net46" />
-  <package id="Microsoft.VisualStudio.Utilities" version="14.3.25407" targetFramework="net46" />  
+  <package id="Microsoft.VisualStudio.Settings.15.0" version="15.0.25824-RC" targetFramework="net46" />
+  <package id="Microsoft.VisualStudio.Utilities" version="14.3.25407" targetFramework="net46" />
   <package id="Microsoft.VisualStudio.Language.StandardClassification" version="14.3.25407" targetFramework="net46" />
-  <package id="Microsoft.VisualStudio.Language.Intellisense" version="14.3.25407" targetFramework="net46" />  
+  <package id="Microsoft.VisualStudio.Language.Intellisense" version="14.3.25407" targetFramework="net46" />
```

Comment:
Do we need to update these references in any other `packages.config` in the rep?